app/views/layouts/_sidebar.html.erb

```diff
@@ -211,3 +211,33 @@
   <% end %>
 <% end %>
 
+<%# Change to true to enable queues %>
```

Comment:
Should this be SCC only? If so it should be inside of the `if` block above (which ends at like 212)